ocfweb/docs/docs/staff/backend/rt.md

```diff
@@ -0,0 +1,57 @@
+[[!meta title="Request Tracker"]]
+
+[**Request Tracker**](https://rt.ocf.berkeley.edu/) is the ticketing system
+used by the OCF. It is the main way of keeping track of OCF-related activity.
+Tickets are currently made either by staff to propose things to talk about
+or by emails from the Hostmaster or Help mailing lists.
```

Comment:
Or security, bod, etc.

I'd probably say "Tickets are automatically created when emails are received at the queue's name, e.g. help@ocf.berkeley.edu. Staff can also create tickets by logging in to the web UI".